Application/Business/LevelServices/LevelServices.cs

```diff
@@ -62,8 +54,16 @@ public async Task CheckLevel(LevelEntity level)
         {
             if(level.Id == Guid.Empty || levelCheck.Id != level.Id)
             {
-                throw new BadRequestException("The game already has this level's name");
+                throw new BadRequestException("The user already have a character in this game server");
             }
         }
     }
+    public async Task CheckLevel(Guid id)
```

Comment:
so many duplicated logics, how about create a function FoundOrThrow(...args) in repo and reuse it?

Reply:
Add new function in Generic Repository:
```
    public virtual async Task CheckExistAsync(Guid id, string message, params string[] navigationProperties)
    {
        var query = ApplyNavigation(navigationProperties);
        T entity = await query.FirstOrDefaultAsync(e => e.Id.Equals(id));
        if(entity is null)
        {
            throw new NotFoundException(message);
        }
    }
```
Remove all Check... functions in services